lib/config/httpRoutes.js

```diff
@@ -88,16 +88,17 @@ const routes = [
   { verb: 'get', path: '/profiles/_scroll/:scrollId', controller: 'security', action: 'scrollProfiles' },
 
   { verb: 'get', path: '/_adminExists', controller: 'server', action: 'adminExists' },
-  { verb: 'get', path: '/_getAllStats', controller: 'server', action: 'getAllStats' },
+  { verb: 'get', path: '/_getAllStats', controller: 'server', action: 'getAllStats', deprecated: { since: '2.15.0', message: 'Use this route instead: http://kuzzle:7512/_metrics' } }, // @deprecated
```

Comment:
Wrong version number. You should use the `auto-version` tag instead which is replaced during the release process. Also the `// @deprecated` comment is redundant IMHO

Reply:
Also, the API documentation should be updated with the `DeprecatedBadge` as well